src/generic/configure-modal/ConfigureModal.test.jsx

```diff
@@ -390,16 +390,152 @@ describe('<ConfigureModal /> for Unit', () => {
   });
 
   it('disables the Save button and enables it if there is a change', () => {
-    useSelector.mockReturnValue(
+    const newCurrentItemData = {
+      ...currentUnitMock,
+      userPartitionInfo: {
+        ...currentUnitMock.userPartitionInfo,
+        selectedPartitionIndex: 0,
+      },
+    };
+    const { getByRole, getByTestId } = renderUnitComponent({
+      currentItemData: newCurrentItemData,
+    });
+
+    const saveButton = getByRole('button', { name: messages.saveButton.defaultMessage });
+    expect(saveButton).toBeDisabled();
+
+    const input = getByTestId('group-type-select');
+    // unrestrict access
+    fireEvent.change(input, { target: { value: -1 } });
+    expect(saveButton).not.toBeDisabled();
+
+    fireEvent.change(input, { target: { value: 0 } });
```

Comment:
can we `userEvent` here?

Reply:
Do we need to rewrite all tests because it isn't our code, and we will need an extra time for it?